### PR TITLE
Issue #643 Document troubleshooting expired certificates

### PR DESCRIPTION
**Pull request by robin-owen**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -11,11 +11,10 @@ Using an older `crc` binary will result in an expired certificates error.
 
 To resolve this error:
 
-* link:https://github.com/code-ready/crc/releases[Download the latest CodeReady Containers release] and place the `crc` binary in your `$PATH`.
+* link:https://cloud.redhat.com/openshift/install/crc/installer-provisioned[Download the latest CodeReady Containers release] and place the `crc` binary in your `$PATH`.
```
Comment (praveenkumar) [inline README.adoc]: :+1:

File: docs/source/getting-started/content/proc_troubleshooting-expired-certificates.adoc [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+. Remove the cluster with certificate errors using the [command]`{bin} delete` command:
++
+[subs="+quotes,attributes"]
+----
+$ {bin} delete
+----
++
+[NOTE]
+====
+The [command]`{bin} delete` command will result in the loss of data stored in the {prod} virtual machine.
```
Comment (praveenkumar) [inline docs/source/getting-started/content/proc_troubleshooting-expired-certificates.adoc]: :+1:

File: docs/source/getting-started/content/proc_troubleshooting-expired-certificates.adoc [hunk 3]
```diff
@@ -0,0 +1,37 @@
+[id="troubleshooting-expired-certificates_{context}"]
+= Troubleshooting expired certificates
+
+Each released `{bin}` binary expires 30 days after the release due to certificates embedded in the OpenShift cluster.
+Using an older `{bin}` binary will result in an expired certificates error.
```
Comment (cfergeau) [inline docs/source/getting-started/content/proc_troubleshooting-expired-certificates.adoc]: This sentence is misleading, at the moment we force crc binaries to be used with a given bundle version, but it really is using the older *bundle* which has the certificate issues.
If it was possible to run latest binary with an older bundle, we'd still get certificate issues.

Comment (robin-owen) [reply docs/source/getting-started/content/proc_troubleshooting-expired-certificates.adoc]: Good point, will update soon. Thanks!

File: docs/source/getting-started/content/proc_troubleshooting-expired-certificates.adoc [hunk 4]
```diff
@@ -0,0 +1,37 @@
+[id="troubleshooting-expired-certificates_{context}"]
+= Troubleshooting expired certificates
+
+Each released `{bin}` binary includes an embedded system bundle that expires 30 days after the release due to certificates embedded in the OpenShift cluster.
```
Comment (robin-owen) [inline docs/source/getting-started/content/proc_troubleshooting-expired-certificates.adoc]: @cfergeau, is this explanation clearer?

Comment (cfergeau) [reply docs/source/getting-started/content/proc_troubleshooting-expired-certificates.adoc]: This is good for me, thanks!

**Pull request merged**